docs/configuration/required-settings.md

```diff
@@ -21,6 +21,7 @@ NetBox requires access to a PostgreSQL database service to store data. This serv
 * `PASSWORD` - PostgreSQL password
 * `HOST` - Name or IP address of the database server (use `localhost` if running locally)
 * `PORT` - TCP port of the PostgreSQL service; leave blank for default port (5432)
+* `CONN_MAX_AGE` - Number in seconds for Netbox to keep database connections open. 150-300 seconds is typically a good starting point ([more info](https://docs.djangoproject.com/en/2.2/ref/databases/#persistent-connections)).
```

Comment:
One trick with Django docs: you can use `stable` in place of the version number and it will automatically redirect to the current stable release (e.g. https://docs.djangoproject.com/en/stable/ref/databases/#persistent-connections). This helps prevent links from becoming outdated.

Reply:
I'll fix that. The other Django docs links that I saw in these files were referencing version specific URLs so I followed along.

Reply:
That reminds me, I want to fix those.